Stratis.Bitcoin/Features/MemoryPool/MempoolFeature.cs

```diff
@@ -13,7 +15,7 @@ namespace Stratis.Bitcoin.Features.MemoryPool
     /// Transaction memory pool feature for the Full Node.
     /// </summary>
     /// <seealso cref="https://github.com/bitcoin/bitcoin/blob/6dbcc74a0e0a7d45d20b03bb4eb41a027397a21d/src/txmempool.cpp"/>
-    public class MempoolFeature : FullNodeFeature, IFeatureStats
+    public class MempoolFeature : FullNodeFeature, IFeatureStats, IPooledTransaction
```

Comment:
This does not have to go on the MempoolFeature , it can go on the 'mempoolManager'
We might want to try to keep the features clean just for start/stop/registration/validation.